cypress/integration/apps/citas/agendas/revision_agendas.spec.js

```diff
@@ -0,0 +1,98 @@
+/// <reference types="Cypress" />
+
+const agenda = require('../../../../fixtures/agenda_auditada');
+
+context('TM Profesional', () => {
```

Comment:
Cambiar el nombre del test.
```suggestion
context('CITAS - Revision de agendas', () => {
```